Files.md

```diff
@@ -0,0 +1,41 @@
+
+
+This page documents the different files in this repository.  It is meant for beginners who do not quite know what is going on. 
+
+### .vscode 
+This is used by Microsoft Visual Studio.  If you are not using Visual Studio you can ignore it.  If you are using Visual Studio, I would be grateful if you said a word about these files. 
```

Comment:
It's actually [Visual Studio _Code_](https://code.visualstudio.com/), not [Visual Studio](https://visualstudio.microsoft.com/). They are two seperate things.